tests/test_model_selection/test_importances.py

```diff
@@ -407,10 +409,6 @@ def test_is_fitted(self):
         del visualizer.features_
         assert not visualizer._is_fitted()
 
-    @pytest.mark.xfail(
-        reason="""third test fails with AssertionError: Expected fit
-        to be called once. Called 0 times."""
-    )
     def test_with_fitted(self):
```

Comment:
So I guess this works now!? 🎉

Reply:
Indeed 😎 ! We still have a [few more](https://github.com/DistrictDataLabs/yellowbrick/search?q=third+test+fails+with+AssertionError&unscoped_q=third+test+fails+with+AssertionError) of these to figure out, but it seems likely they'll be mostly addressed with #939!